src/executables/uber-time.js

```diff
@@ -8,13 +8,19 @@ import CommandExecutionService from '../services/CommandExecutionService';
 
 const service = new CommandExecutionService();
 
+const address = program.args[0].trim().toString();
```

Comment:
It helps when a user puts in something like a zipcode which would normally be rejected. I also modified the test so it fails on invalid or non-string convertable characters, but mostly everything gets string typed and then passed through.